### PR TITLE
Add support for Scala with ScalaPBC

### PR DESCRIPTION
**Pull request by spoonscen**
This PR adds support for scala using the ScalaPBC as a protoc plugin.
https://scalapb.github.io/scalapbc.html

It also rolls back the version bump that is currently breaking the master build.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -56,8 +57,14 @@ RUN make -C /go/src/github.com/envoyproxy/protoc-gen-validate/ build
 
 RUN go get -u github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
 
-# Add grpc-web support
 
+
+# Add scala support
```
Comment (spoonscen) [inline Dockerfile]: Do we want to make this one `RUN` command? I saw both multiple `RUN` commands as well as `&&` commands in the `Dockerfile`.

**Comment by aldelucca1** [comment]
Thanks for this, sorry for the broken master, all is good now, if you wouldn’t mind rebasing your change onto the current master it’d be greatly appreciated

File: Dockerfile (hunk 1 above)
Comment (aldelucca1) [reply Dockerfile]: Yeah if we could collapse this to a single RUN it will generate less layers

**Comment by spoonscen** [comment]
@aldelucca1 Thanks for the review! I rebased and collapsed into a single `RUN`.

**Comment by spoonscen** [comment]
@aldelucca1 Anything else you need from me to get this moved along? Thanks!

**Comment by spoonscen** [comment]
🎉

**Pull request merged**